src/connector/src/parser/maxwell/simd_json_parser.rs

```diff
@@ -128,8 +122,8 @@ mod tests {
     #[tokio::test]
     async fn test_json_parser() {
         let descs = vec![
-            SourceColumnDesc::simple("ID", DataType::Int32, 0.into()),
-            SourceColumnDesc::simple("NAME", DataType::Varchar, 1.into()),
+            SourceColumnDesc::simple("id", DataType::Int32, 0.into()),
+            SourceColumnDesc::simple("name", DataType::Varchar, 1.into()),
```

Comment:
the test is intentional. for the mismatch between column name and json keys

Reply:
Do you mean that we need both `match` and `mismatch` test cases?

Without the changes, https://github.com/risingwavelabs/risingwave/pull/9106/files#diff-9ba8fcf927d4f456ea34620beb6dfe24ffe28683b2f6b40bd8e2aa76a1955377R153 currently cannot pass

Reply:
https://github.com/risingwavelabs/risingwave/pull/9086 should handle such cases, column name `Name`, `NAme` should be able to match `name` in json